src/main/resources/edu/cornell/kfs/coa/document/datadictionary/SubObjectCodeMaintenanceDocument.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    The Kuali Financial System, a comprehensive financial management system for higher education.
+
+    Copyright 2005-2021 Kuali, Inc.
```

Comment:
Does this file need the KualiCo license header? It's not overlaying the related file from base code.